sysutils/pfSense-pkg-Service_Watchdog/files/usr/local/pkg/servicewatchdog.inc

```diff
@@ -106,7 +106,12 @@ function servicewatchdog_check_services() {
 	}
 	$a_pwservices = &$config['installedpackages']['servicewatchdog']['item'];
 
-	foreach ($a_pwservices as $svc) {
+	foreach ($a_pwservices as $idx => $svc) {
+		// apinger became dpinger in pfSense 2.3
+		if ($svc['name'] == 'apinger') {
+			$a_pwservices[$idx]['name'] = $svc['name'] = 'dpinger';
+			write_config(gettext("Service Watchdog updated service apinger to dpinger"));
```

Comment:
I made PR #106 which has the simplified version without write_config() - take your pick.